src/entity/units/bloodGolem.ts

```diff
@@ -16,15 +16,15 @@ const unit: UnitSource = {
   },
   unitProps: {
     damage: 40,
-    staminaMax: config.UNIT_BASE_STAMINA * 1.2,
-    healthMax: 60,
+    staminaMax: config.UNIT_BASE_STAMINA,
+    healthMax: 80,
     manaMax: 0,
     bloodColor: 0x8a2e2e,
   },
   spawnParams: {
     probability: 100,
-    budgetCost: 4,
-    unavailableUntilLevelIndex: 4,
+    budgetCost: 10,
+    unavailableUntilLevelIndex: 3,
```

Comment:
This feels too early to show an upgraded version of the regular golem

Reply:
It feels like it's time to ramp up once you get to the lava levels. HP changes make midgame kind've easy, and this is supposed to help with that